misk-admin/src/main/kotlin/misk/web/metadata/config/ConfigMetadataAction.kt

```diff
@@ -51,20 +74,10 @@ class ConfigMetadataAction @Inject constructor(
 
     fun redact(output: String, regex: Regex) =
       output.replace(regex, "████████")
+  }
 
-    fun generateConfigResources(
-      appName: String,
-      deployment: Deployment,
-      config: Config
-    ): Map<String, String?> {
-      val rawYamlFiles = MiskConfig.loadConfigYamlMap(appName, deployment, listOf())
-      val yamlFiles = linkedMapOf<String, String?>(
-        "Effective Config" to MiskConfig.toYaml(
-          config, ResourceLoader.SYSTEM
-        )
-      )
-      rawYamlFiles.map { yamlFiles.put(it.key, it.value) }
-      return yamlFiles
-    }
+  enum class ConfigTabMode {
+    SECURE, // Only show secure content which will not leak Misk secrets
```

Comment:
maybe `SAFE`/`UNSAFE_LEAK_MISK_SECRETS` would be clearer? hopefully we can fix the underlying secrets issue before too long so this doesn't much matter.